src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
 // write code here
+const headers = document.querySelectorAll('th');
+
+const sortBy = (columnIndex) => {
+  const table = document.querySelector('table');
+  const rows = Array.from(table.rows);
+  const rowsToSort = rows.slice(1, rows.length - 1);
```

Comment:
The slicing of `rows` to create `rowsToSort` is incorrect. The current implementation `rows.slice(1, rows.length - 1)` excludes the last row from being sorted. If you intend to sort all rows except the header, use `rows.slice(1)` instead.